style.css

```diff
@@ -0,0 +1,111 @@
+body, html{
+    background-color: #4682b4;
+    margin: 0;
+    color: white;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+header{
+    background-image: linear-gradient(#d1eaf0, #012a4a);;
+    position: fixed;
+    width: 100%;
+}
+nav li {
+    list-style: none;
+    font-size: 20px;
+}
+.navbar{
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+}
+a,
+a:link {
+    color:white;
+    text-decoration: none;}
+h1{
+    font-size: 45px;
+    text-align: center;
+}
+#home ul li{
+    list-style: none;
+    }
+#home{
+    color: white;
+    text-align: center;
+    background-image: url(photos/sd2.jpg);
+    background-position: center;
+    width: 100%;
+    height: 59rem;
+}
+.home-container{
+    padding: 15rem;
+}
+.quote{
+    font-size: xx-large;
+    background-color: #0077b6;
+    border:none;
+    border-radius: 10px;
+    box-shadow: 8px 5px 5px whitesmoke;
+}
+#bio{
+    margin: 15rem;
+    padding-top: 6rem;
+    height: 60rem;
+}
+#bio img{
+    height:500px;
+    justify-content: space-around;
+}
+.biophotos{
+    display: flex;
+    justify-content: space-around;
+}
+#bio-info{
+    border:dashed blue;
+    padding:25px;
+    height: auto;
+    font-size: 20px;
+}
+#experience{
+    padding-top: 5rem;
+    background-image: url(photos/sd1.jpg);
+    background-position: center;
+    height: 59rem;
+}
+#jobs{
+    display: flex;
+    justify-content: center;
+}
+.white{
+    width: 25%;
+    background-color: rgba(0,0,0,0.5);
+    border:solid #014f86;
+    color: white;
+    margin: 2rem;
+    padding: 1rem;
+    text-align: center;
+}
+.ja,
+.hertz,
+.pimc,
+.sante,
+.fvc{
```

Comment:
love this condensing of several classes using the same set of CSS properties and values!